package.json

```diff
@@ -87,12 +87,14 @@
     "gulp-cli": "^2.0.1",
     "gulp-execa": "^2.0.0",
     "mocha": "^7.0.1",
+    "react-sweet-progress": "^1.1.2",
     "rollup": "^1.1.0",
     "rollup-plugin-babel": "^4.3.3",
     "rollup-plugin-commonjs": "^10.1.0",
     "rollup-plugin-copy": "^3.3.0",
     "rollup-plugin-json": "^4.0.0",
     "rollup-plugin-node-resolve": "^5.2.0",
+    "rollup-plugin-postcss": "^3.1.2",
```

Comment:
I'm assuming this was needed to import `react-sweet-progress/lib/style.css`, right?